dobby/io/perkin_elmer.py

```diff
@@ -0,0 +1,210 @@
+            self.computed_tidy = self.tidify(self.computed_data,
+                                             'computed_values')
+
+            # Add computed percent viability to tidy
+            self.tidy = self.tidy.merge(self.computed_tidy,
+                                        on=self.merge_columns)
+
+
+    @staticmethod
+    def tidify(data2d, value='value'):
+        """Convert 2d raw_data matrix to tidy format, each row is 1 observation"""
+        data2d.index.name = 'row_letter'
+        data2d.columns.name = 'column_number'
+        tidy = data2d.unstack().reset_index()
+        tidy = tidy.rename(columns={0: value})
+        tidy['well'] = tidy['row_letter'] + tidy['column_number']
+        return tidy
+
+    def merge_plate_data_and_map(self):
+
+        self.tidy = self.map_tidy.merge(self.rawdata_tidy,
+                                        on=self.merge_columns,
+                                        suffixes=['_map', '_data'])
+
+
+
+    def __repr__(self):
+        return f"Plate measured at {self.measurement_date}"
+
+    def plot(self, ax=None, raw=True):
+        if ax is None:
+            fig, ax = plt.subplots(figsize=(12, 6))
+        if raw:
+            data = self.raw_data
+            cmap = self.raw_cmap
+        else:
+            data = self.computed_data
+            cmap = self.computed_cmap
+
+        sns.heatmap(data, annot=self.map, fmt='', ax=ax, cmap=cmap)
+
+    @property
+    def _tidy(self):
+        """Internal method for tidy raw_data that deals with edge effects"""
+        if self.ignore_edge_rows:
+            tidy = self.tidy.query(
+                '(row_letter not in @self.edge_letters)')
+        else:
+            tidy = self.tidy
+        return tidy
+
+    @property
+    def controls(self):
+        return self._tidy.query('plate_map == @self.control')
+
+    @property
+    def edge_letters(self):
+        return self.raw_data.index[0], self.raw_data.index[-1]
+
+    def mean(self):
+        return self.controls['raw_values'].mean()
+
+    def stdev(self):
+        return self.controls['raw_values'].std()
+
+    def cv(self):
+        return 100 * self.stdev() / self.mean()
+
+
+
+def reader(path):
+    with open(path, newline='') as csv_file:
+        out = {}
+        csv_reader = csv.reader(csv_file, delimiter=',')
+        for idx, row in enumerate(csv_reader):
+            if len(row) != 0:
+                # Filter out empty strings with 'if x'
+                out[idx] = [x.strip() for x in row if x]
+    return out
+
+def parse(path, plate_map, nrow_per_plate=8, plot=True):
+    lines = reader(path)
+    plates = []
+
+    plate_kwargs = None
+    background_kwargs = None
+    data = None
+    for key, value in lines.items():
+
+        # Get plate layout information
+        if value[0] == 'Plate information':
+            keys = map(sanitize, lines[key + 1])
+            arguments = [x if x != 'N/A' else np.nan for x in lines[key + 2]]
```

Comment:
This is a nit and mostly a preference thing, but personally I would avoid the double-negative here and elsewhere and just go with
```suggestion
            arguments = [np.nan if x == 'N/A' else x for x in lines[key + 2]]
```